### DISCUSSION TITLE
简报: Bilibili HTML5播放器 V4.0.2 已发布更新日志

### DISCUSSION THREAD

**Discussion by Tinhone** [the1812/Bilibili-Evolved#4104] [综合讨论 / General]
感觉很快就会全量推送，我这里已经被更新到 V4 了，不过完全看不出来更新了什么，个人安装的所有组件目前还运行正常

V2 大版本最后一次出现在更新日志上是 2022-08-02 更新的 V2.82.0，距离现在也才七个月。V4 这么快推出，完全搞不明白是在干啥...

https://www.bilibili.com/blackboard/webplayer_history.html#html5

![图片](https://user-images.githubusercontent.com/81268244/228213873-764265ec-b6d6-47c5-ad54-1f8c1fb71fcb.png)

**Comment by the1812** [comment]
每次就是改改class，换几个更垃圾的CDN给你（

**Comment by Tinhone** [comment]
奇怪，又变回 V3 了
